tools/claude/plugins/dxos/hooks/track.sh

```
@@ -91,6 +94,18 @@ emit_track() {
   emit "$(printf 'TASK-PLANNING DIRECTIVE: `/dxos:project track`. Record this follow-up in the TASKS.md of the current unit of work (package or directory) per the task-planning skill — do NOT use a background task chip. Item: "%s". Confirm in one short line.' "$1")"
 }
 
+emit_history() {
+  emit "$(printf 'TASK-PLANNING PROJECT DIRECTIVE: `/dxos:project history` — args: "%s". Report the PRs a project has produced, newest first. Default to the CURRENT project (resolve it exactly as for bare `/dxos:project`); args `all` covers every project belonging to the current user (`whoami`); a bare name restricts to that project. The registry`s `prs` list is the SOURCE OF TRUTH for which PRs count as tracked — never enumerate the branch`s commits or the repo`s PR list instead, since those include work this project never claimed. Enrich each number with `gh pr view <n> --json number,title,author,mergedAt,createdAt,state` (one call per PR, or `gh pr list` filtered if that is fewer round-trips). Render ONE markdown table, columns: # | PR (as a markdown link to the PR URL) | date (merged date, else created, ISO yyyy-mm-dd) | user (the GitHub author login) | summary. The summary is ONE sentence in your own words describing what the PR changed — derive it from the title and the project ledger, never paste the raw title if it is uninformative. Add a `project` column only when covering more than one project. If `gh` is unavailable or unauthenticated, say so in one line and still render the table with the PR numbers and whatever the registry knows, leaving date and user blank rather than guessing. If the project has no `prs`, say so in one line and stop.' "$1")"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)